### DISCUSSION TITLE
Hindi Language Leader

### DISCUSSION THREAD

**Discussion by timaseth** [OHF-Voice/intents#783] [Language Leaders]
I am happy to contribute in Hindi.
I am raw with pulls and pushes but will try to be of use :-)

Cheers

**Comment by arunshekher** [comment]
@timaseth there is already [a thread created for Hindi language discussions](https://github.com/home-assistant/intents/discussions/363). I think its better to post there.

Read [this guideline](https://developers.home-assistant.io/docs/voice/language-leaders/) and fulfill the requirements to become a language leader.

Make 2+ pull-requests to the repository to get started.

You may do it via 3 ways:

1. [Github Edit](https://github.com/home-assistant/intents/blob/main/docs/github/README.md)
2. [Github Codespace](https://github.com/home-assistant/intents/blob/main/docs/codespace/README.md)
3. [Local Fork](https://github.com/home-assistant/intents/blob/main/docs/forking/README.md)

[Contributing guidelines can be found here.](https://developers.home-assistant.io/docs/voice/intent-recognition/contributing/)
